src/DocumentationBrowserViewExtension/DocumentationBrowserView.xaml.cs

```diff
@@ -136,7 +138,11 @@ async void InitializeAsync()
             {
                 if (viewModel.Link != null && !string.IsNullOrEmpty(viewModel.CurrentPackageName))
                 {
-                    VirtualFolderPath = Path.GetDirectoryName(HttpUtility.UrlDecode(viewModel.Link.AbsolutePath));
+                    string absolutePath = Path.GetDirectoryName(HttpUtility.UrlDecode(viewModel.Link.AbsolutePath));
+                    //We move two levels up so it will be located in same level than the the fallback_docs_images directory
```

Comment:
fallback_docs_images  -> NodeHelpDocsImages

Reply:
please use `var`

Reply:
changed to use var, see commit:
https://github.com/DynamoDS/Dynamo/pull/14429/commits/ef88df024b2dee733e26ff26f7c81188bdab999f